src/api/app/views/webui2/webui/groups/_group_members.html.haml

```diff
@@ -16,20 +16,19 @@
     .d-flex.flex-wrap
       - group.users.each do |user|
         .card.m-1.p-2.group-user
+          - if write_access
+            = link_to(group_user_path(group_title: group.title, user_login: user.login), data: { method: :delete, remote: true },
+              id: user.login, class: 'group-member-removal d-inline text-right ', title: 'Remove member from group') do
```

Comment:
It's a nice solution :+1:

We can remove the class `d-inline` since a link is already an inline element.

Reply:
Of course! I only forgot to delete it after testing different things.